assignments/prototypes.js

```diff
@@ -32,7 +52,16 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+function Humanoid(hooman){
+  CharacterStats.call(this, hooman);
+  this.team = hooman.team;
+  this.weapons= hooman.weapons;
+  this.language= hooman.language;
+}
+
```

Comment:
And this gives Humanoid the CharacterStats prototypal inheritance. 

```suggestion

Humanoid.prototype = Object.create(CharacterStats.prototype);

```